src/commands/math/commands.js

```diff
@@ -191,7 +191,12 @@ var SupSub = P(MathCommand, function(_, super_) {
         else cursor.clearSelection().insRightOf(this.parent);
         return cmd.createLeftOf(cursor.show());
       }
-      if (cursor.options.charsThatBreakOutOfSupSub.indexOf(ch) > -1) {
+      if (
+        cursor.options.charsThatBreakOutOfSupSub.indexOf(ch) > -1 &&
+        // Unary operators never break out of exponents, so that it's easy to
+        // write negative exponents.
+        cursor[L] !== 0
```

Comment:
Is there a better way to check whether an operator is being used as a unary prefix operator than to check whether the cursor is at the beginning of the block?